miner/worker.go

```diff
@@ -571,12 +585,12 @@ func (miner *Miner) commitTransactions(env *environment, plainTxs, blobTxs *tran
 		}
 
 		// OP-Stack addition: Jovian DA footprint limit
-		var txDAFootprint *big.Int
+		var txDAFootprint uint64
 		// Note that commitTransaction is only called after deposit transactions have already been committed,
 		// so we don't need to resolve the transaction here and exclude deposits.
 		if isJovian {
-			txDAFootprint = new(big.Int).Mul(ltx.DABytes, big.NewInt(params.DAFootprintGasScalar))
-			if daFootprintLeft.Cmp(txDAFootprint) < 0 {
+			txDAFootprint = ltx.DABytes.Uint64() * uint64(env.daFootprintGasScalar)
+			if daFootprintLeft < txDAFootprint {
```

Comment:
You're changing it to `uint64` here, but why not below for `daBytesAfter` also?

And if everything can be `uint64`, can that unify the `env.daFootprint` and `blockDABytes` accounting?